frontend/src/views/DocumentListView.vue

```diff
@@ -0,0 +1,150 @@
+<script setup lang="ts">
+import { saveAs } from "file-saver";
+import Button from "primevue/button";
+import Column from "primevue/column";
+import DataTable from "primevue/datatable";
+import InputText from "primevue/inputtext";
+import { onMounted, ref, watch } from "vue";
+
+import { deleteDocument, getDocumentPdf, getDocuments } from "@/backendClient";
+import useConfirmations from "@/compositions/useConfirmations";
+import useNotifications from "@/compositions/useNotifications";
+import { DocumentKind } from "@/global/types/appTypes";
+import type {
+  InvoiceDocument,
+  OfferDocument,
+  OverdueNoticeDocument,
+} from "@/global/types/entities";
+import { debounce } from "@/helpers/utils";
+import { ROUTES } from "@/router";
+
+const confirm = useConfirmations();
+const notifications = useNotifications();
+const documentsList = ref<(InvoiceDocument | OfferDocument | OverdueNoticeDocument)[]>([]);
+const search = ref<string>("");
+const selectedProduct = ref<(OfferDocument | OverdueNoticeDocument | InvoiceDocument)[]>([]);
+
+function getDocumentType(doc: OfferDocument | OverdueNoticeDocument | InvoiceDocument) {
+  if ("offer_id" in doc) {
+    return DocumentKind.offer;
+  } else if ("overdue_notice_id" in doc) {
+    return DocumentKind.overdueNotice;
+  } else {
+    return DocumentKind.invoice;
+  }
+}
+
+async function reloadPage() {
+  // TODO: use pagination
+  documentsList.value = await getDocuments(search.value);
+}
+
+async function removeDocument(doc: OfferDocument | OverdueNoticeDocument | InvoiceDocument) {
+  let kind = getDocumentType(doc);
+  await deleteDocument(doc.id, kind);
+  reloadPage();
+}
+
+function getSelectedIds() {
+  return selectedProduct.value.map(function (element) {
+    return element.id;
+  });
+}
+
+async function createPdf() {
+  let selectedValuesArray = selectedProduct.value.map(function (element) {
+    return { kind: getDocumentType(element), id: element.id };
+  });
+  const response = await getDocumentPdf(selectedValuesArray);
+  notifications.showCreatePdfDocumentNotification();
+  saveAs(response, `${getSelectedIds().join("-")}.pdf`);
+}
+
+const confirmCreatePdf = () => {
+  confirm.showCreatePdfConfirmation(createPdf, getSelectedIds());
+};
+
+watch(search, debounce(reloadPage, 250));
```

Comment:
When I change the search value nothing updates for me. Is this the same for you?

Reply:
<img width="802" alt="image" src="https://github.com/user-attachments/assets/4d0c3fd1-77f1-439d-a49e-637cebcf86bf">
The search works on the frontend and makes a request, but something is wrong with the backend.